biomage/experiment/download.py

```diff
@@ -0,0 +1,338 @@
+import json
+import os
+from enum import Enum
+from fileinput import filename
+from pathlib import Path
+
+import boto3
+import click
+from botocore.exceptions import ClientError
+
+from ..rds.run import run_rds_command
+from ..utils.constants import (CELLSETS_BUCKET, PROCESSED_RDS_BUCKET,
+                               RAW_RDS_BUCKET, SAMPLES_BUCKET, SAMPLES_TABLE)
+from .utils import set_modified_date
+
+output_path = "."
+
+
+s3 = boto3.resource("s3")
+SAMPLES = "samples"
+RAW_RDS = "raw_rds"
+PROCESSED_RDS = "qc_rds"
```

Comment:
can you change these names to `RAW_FILES` and `PROCESSED_FILES`?

Reply:
I'll rename it to `RAW_SEURAT_FILE` and `PROCESSED_SEURAT_FILE`

Reply:
I'm going with Iva's suggestion